src/utils/bindActionCreators.js

```diff
@@ -21,6 +22,13 @@ function bindActionCreator(actionCreator, dispatch) {
  * action creator wrapped into the `dispatch` call.
  */
 export default function bindActionCreators(actionCreators, dispatch) {
+  invariant(typeof actionCreators === 'function' ||
```

Comment:
Good call moving it down. You could check if the actionCreators is NOT an object with `typeof actionCreators === 'object'`? That way undefined, null and primitives are all caught.